package.json

```diff
@@ -143,7 +143,8 @@
   "main": "Parse-Dashboard/app.js",
   "jest": {
     "roots": [
-      "src/lib"
+      "src/lib",
+      "Parse-Dashboard"
```

Comment:
The tests are actually under `/src/lib/tests`. Could you move the test to there?

I suspect that you can actually move the test into one of the existing files instead of creating a new file, if it fits thematically:
- `src/lib/tests/e2e/dashboard.e2e.test.js` or 
- `src/lib/tests/e2e.js`

Reply:
I moved the test to src/lib/tests/e2e/dashboard.e2e.test.js.